src/components/Collection/Collection.tsx

```diff
@@ -0,0 +1,245 @@
+        <Hero
+          backgroundColor="section.research.primary"
+          heroType="tertiary"
+          heading={
+            <Heading level="one" id="tertiary-hero">
+              <>
+                Digital Research Books <sup>Beta</sup>
+              </>
+            </Heading>
+          }
+        />
+      </TemplateBreakout>
+
+      <TemplateContent>
+        <TemplateContentTop>
+          <Heading size="primary" marginBottom="xl">
+            {`Collection - ${title}`}
+          </Heading>
+          <Heading size="secondary" marginBottom="l">
+            About this collection
+          </Heading>
+          <Box>{description}</Box>
+        </TemplateContentTop>
+        <TemplateContentPrimary>
+          <HorizontalRule bg="section.research.primary" marginBottom="xl" />
+          <Heading size="secondary">In this collection</Heading>
+          <Flex justify="space-between" marginBottom="xl" align="center">
+            <Heading level="two" size="tertiary" noSpace>
+              {totalItems > 0
+                ? `Viewing ${firstElement.toLocaleString()} - ${
+                    totalItems < lastElement
+                      ? totalItems.toLocaleString()
+                      : lastElement.toLocaleString()
+                  } of ${totalItems.toLocaleString()} items`
+                : "Viewing 0 items"}
+            </Heading>
+            <Form id="results-sorts-form">
+              <ResultsSorts
+                perPage={currentCollectionQuery.perPage}
+                sort={getSortValue()}
+                sortMap={collectionSortMap}
+                onChangePerPage={(e) => onChangePerPage(e)}
+                onChangeSort={(e) => onChangeSort(e)}
+              />
+            </Form>
+          </Flex>
+          <SimpleGrid columns={1} gap="l">
+            {publications.map((pub, c) => {
+              return (
+                <Box key={`collection-item-${c}`}>
+                  <Heading level="two" marginBottom="xs">
+                    <Link
+                      to={{
+                        pathname: pub.links.find(
+                          (link) => link.rel === "alternate"
+                        ).href,
+                      }}
+                      className="link link--no-underline"
+                    >
+                      {truncateStringOnWhitespace(
+                        pub.metadata.title,
+                        MAX_TITLE_LENGTH
+                      )}
+                    </Link>
+                  </Heading>
+                  {pub.metadata.subtitle && (
+                    <Box marginBottom="xs">{pub.metadata.subtitle}</Box>
+                  )}
+                  {pub.metadata.creator && (
+                    <Box marginBottom="xs">
+                      By <Author author={pub.metadata.creator} />{" "}
+                    </Box>
+                  )}
+                  <CollectionItemCard collectionItem={pub}></CollectionItemCard>
+                </Box>
```

Comment:
Since this component is quite large and complex, let's pull this out into a separate component and put it in a new file.